items/src/test/scala/weco/api/items/fixtures/ItemsApiGenerators.scala

```diff
@@ -30,43 +34,63 @@ trait ItemsApiGenerators
     with ItemsGenerators {
   this: Suite =>
 
-  def withItemUpdateService[R](
+  def withSierraItemUpdater[R](
     responses: Seq[(HttpRequest, HttpResponse)] = Seq()
-  )(testWith: TestWith[ItemUpdateService, R]): R =
+  )(testWith: TestWith[ItemUpdater, R]): R =
     withMaterializer { implicit mat =>
       withSierraService(responses) { sierraService =>
-        testWith(new ItemUpdateService(sierraService))
+        testWith(new SierraItemUpdater(sierraService))
       }
     }
 
+  def withItemUpdateService[R](
+    itemUpdaters: List[ItemUpdater]
+  )(testWith: TestWith[ItemUpdateService, R]): R =
+    testWith(new ItemUpdateService(itemUpdaters))
+
+  def buildEntry(
+    sierraItemNumber: SierraItemNumber,
+    deleted: String = "false",
+    suppressed: String = "false",
+    holdCount: Int = 0
+  ) = f"""
+                        |{
+                        |  "id": "${sierraItemNumber.withCheckDigit}",
```

Comment:
```suggestion
                        |  "id": "${sierraItemNumber.withoutCheckDigit}",
```

Item IDs from Sierra don't include the check digit. I know this is copy-pasted from previous code, but the previous code was wrong.